modules/widgets/eu-cookie-law/footer.php

```diff
@@ -0,0 +1,77 @@
+<div
+	class="<?php echo 'negative' === $instance['color-scheme'] ? 'negative ' : '';
+	?>hide-on-<?php echo esc_attr( $instance['hide'] ); ?>"
+	data-hide-timeout="<?php echo intval( $instance['hide-timeout'] ); ?>"
+	id="eu-cookie-law"
+>
+	<form method="post">
+		<?php wp_nonce_field( 'eucookielaw' ); ?>
+		<input type="hidden" name="eucookielaw" value="accept" />
+		<input type="hidden" name="redirect_url" value="<?php echo esc_attr( $_SERVER['REQUEST_URI'] ); ?>" />
+		<input type="submit" value="<?php echo esc_attr( $instance['button'] ); ?>" class="accept" />
+	</form>
+
+	<?php if ( 'default' == $instance['text'] || empty( $instance['customtext'] ) ) {
+		echo $defaults['default-text'];
+		?>
+		<br />
+		<?php
+		_e( 'To find out more, as well as how to remove or block these, see here:', 'jetpack' );
+		echo ' ';
+	} else {
+		echo esc_html( $instance['customtext'] ), ' ';
```

Comment:
One thing that I notice is that you should put `,'  '`  in the brackets, and you should replace the empty space with jetpack textdomain. 
I started this this widget while ago, but I hadn't enough time to complete it. Feel free to use any codeblock you like https://github.com/Automattic/jetpack/compare/master...Stoyan0v:add/eu-cookie-law-widget